.buildkite/scripts/run_standard_models.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+## remove failure for testing
+set -euo pipefail
+
```

Comment:
I like this idea and meant to keep them in one file for the time being as I didn't want to add noise that may cause false failures. Now that we know the pipeline works, I think it is worth exploring to make maintenance easier down the road.

Reply:
Not resolving so that we can keep in PR for later reference.

Reply:
I did a quick assessment and identified that this will be much more of a lift than I would like to incorporate within this release. As such, I created a [FR](https://github.com/fivetran/dbt_fivetran_utils/issues/119) so we can track this in the future and properly plan for this update.